osdsctl/cli/replication.go

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) 2018 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements a entry into the OpenSDS service.
+
+*/
+
+package cli
+
+import (
+	"os"
+
+	"encoding/json"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/opensds/opensds/pkg/utils"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+var replicationCommand = &cobra.Command{
+	Use:   "replication",
+	Short: "manage replications in the cluster",
+	Run:   replicationAction,
+}
+
+var replicationCreateCommand = &cobra.Command{
+	Use:   "create <primary volume id> <secondary volume id>",
+	Short: "create a replication of specified volumes in the cluster",
+	Run:   replicationCreateAction,
+}
+
+var replicationShowCommand = &cobra.Command{
+	Use:   "show <replication id>",
+	Short: "show a replication in the cluster",
+	Run:   replicationShowAction,
+}
+
+var replicationListCommand = &cobra.Command{
+	Use:   "list",
+	Short: "list all replications in the cluster",
+	Run:   replicationListAction,
+}
+
+var replicationDeleteCommand = &cobra.Command{
+	Use:   "delete <replication id>",
+	Short: "delete a replication in the cluster",
+	Run:   replicationDeleteAction,
+}
+
+var replicationUpdateCommand = &cobra.Command{
+	Use:   "update <replication id>",
```

Comment:
Don't user need to input some new replication data so as to update the replicaiton?

Reply:
Some features have not  implemented yet, so I just add some basic information  updating right now.

Reply:
If so, I think you should add some TODO items